.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,55 @@
+name: Benchmark Pull Request
+on:
+  pull_request:
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: snok/install-poetry@v1
+        with:
+          version: 1.1.13
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+
+      - name: Run Benchmarking Script
+        run: |
+          curl https://storage.courtlistener.com/bulk-data/eyecite/tests/ten-percent.csv.bz2 --output benchmark/bulk-file.csv.bz2
+          
+          git clone https://github.com/freelawproject/eyecite.git benchmark/eyecite
+          cp benchmark/benchmark.py benchmark/eyecite/ && cd benchmark/eyecite/
+          poetry install --no-dev
+          poetry run pip install matplotlib pandas tabulate
```

Comment:
cant sneak anything passed you .. part of this is poetry not playing nice with my Mac.  for instance ... and you referenced this before... hypercard doesnt seem to want to play with my apple silicon.

Reply:
also - this change might need a rebase of a small PR to main adding these dependencies in before we continue - because otherwise checking out the main branch will crash

Reply:
Well, this is a particularly interesting PR, so yeah, I'm looking at it closely and thinking a lot about architecture both in this one and in the bigger picture if we build more of these auto-commenters in the future. :)